src/app/BufferedReadCallback.h

```diff
@@ -74,6 +74,7 @@ class BufferedReadCallback : public ReadClient::Callback
     void OnError(CHIP_ERROR aError) override
     {
         mBufferedList.clear();
+        mLastError = CHIP_NO_ERROR;
```

Comment:
Why?

Reply:
sorry, I forgot to push latest change, could you re-review? this has been removed